### ISSUE TITLE
Auto resizing keeps getting larger with each click inside Iframe

### ISSUE THREAD

**Issue by Technorocker** [davidjbradshaw/iframe-resizer#827]
Hello and thank you for this library. I have been pulling my hair out trying to resize my iframe dynamically.  I have implemented this per the getting started and it does resize the iframe. 

The problem is that when the page loads it starts at minimal size and then every click I do inside the iframe makes the size get slightly larger. I can do this indefinitely to fill the iframe to the bottom of the page. I am making a social media website and I have the comments hidden till the post is clicked on. My example post has a few comments below it and when clicked they dont show up till i click repeatedly to make the iframe "grow"

I have checked for CSS styling that might be affecting it size wise but I only have the width set to 100%

**Comment by davidjbradshaw** [comment]
The problem is your CSS is setting an element to be more than 100% of the page height and this creates a loop. Your option are:

- Fix your CSS
- Try différent height calculation methods
- Set the tolerance option

**Comment by davidjbradshaw** [comment]
Sorry just read your post again, that is more weird, I would still try the different height calculation methods though

**Comment by Technorocker** [comment]
Thanks for the reply, I will look over my CSS again and see if something else is affecting it. I'm fairly new to website development and am taking a course thats not responsive so I'm trying to add in features to it as I go.

**Comment by Technorocker** [comment]
So I've been playing around with the code and when I try to use the heightcalculationmethod in the function call its returning undefined. I've tried moving the parent script src tag around from the header to the bottom of the index page and a few other places. The comment section is getting called from a php function. Could that be causing the problem?
I have also tried using the jquery call as well from a separate js file linked to the page but get the same result of undefined. I did motice that its jsut the heightcalculationmethod thats returning undefined, the log and I also added in the enablePublicMethods and those dont return an undefined error in the chrome development console
